src/app/page/Home/Home.js

```diff
@@ -372,14 +374,15 @@ const SignInHeaderComponent = ({ history }) => {
  *
  * @component
  */
-const HomePage = ({ auth, history }) => {
+const HomePage = ({ auth, history, profile }) => {
   const classes = useStyles();
-  const isEmpty = useSelector((state) => state.firebase.auth.isEmpty);
-  const isLoaded = useSelector((state) => state.firebase.auth.isLoaded);
+  if (isLoaded(auth) && !isEmpty(auth) && isLoaded(profile)) {
+    // User.createProfile(auth.uid, auth);
```

Comment:
idk what the goal was here but I had to comment this out cause it was completely breaking the site if you weren't logged in

Reply:
ehm, this is for the user who is logged in without going through the signup, to make sure we always have a profile to go with the system